homeassistant/helpers/deprecation.py

```diff
@@ -292,9 +292,18 @@ def check_if_deprecated_constant(name: str, module_globals: dict[str, Any]) -> A
     return value
 
 
-def dir_with_deprecated_constants(module_globals: dict[str, Any]) -> list[str]:
+def dir_with_deprecated_constants(module_globals: list[str]) -> list[str]:
```

Comment:
```suggestion
def dir_with_deprecated_constants(module_globals_keys: list[str]) -> list[str]:
```

As the type is different to the other two methods and so nobody get confused